server/chat-plugins/wifi.tsx

```diff
@@ -1374,317 +1439,457 @@ function makePageHeader(user: User, pageid?: string) {
-						}
-					} else {
-						giveaway = giveaway as QuestionGiveawayData;
-						buf += `<div class="infobox"><h3 style="text-align:center">Question</h3><hr />`;
-						buf += Utils.html`<strong>Game:</strong> ${gameName[giveaway.game]}, <strong>Giver:</strong> ${giveaway.targetUserID}, <strong>OT:</strong> ${giveaway.ot}, <strong>TID:</strong> ${giveaway.tid}`;
-						if (giveaway.claimed) {
-							buf += Utils.html`<br /><strong>Claimed:</strong> ${giveaway.claimed}`;
-						}
-						buf += Utils.html`<br /><strong>Question:</strong> ${giveaway.question}`;
-						buf += `<br /><strong>Answer${Chat.plural(giveaway.answers.length, "s")}:</strong> ${giveaway.answers.join(', ')}`;
-						buf += `<br /><strong>Pok&eacute; Ball:</strong> <psicon item="${giveaway.ball}" />`;
-						buf += `<details><summary><psicon pokemon="${giveaway.prize.species}" /> Prize</summary>`;
-						buf += `${Chat.formatText(Giveaway.convertIVs(giveaway.prize, giveaway.ivs), true)}</details>`;
-						if (giveaway.extraInfo?.trim()) {
-							buf += `<hr /><details><summary>Extra Info</summary>${Chat.formatText(giveaway.extraInfo.trim(), true)}</details>`;
+			if (!giveaways.length && !adding) {
+				return <div class="pad">
+					{makePageHeader(this.user, args[0] === 'add' ? 'submitted-add' : 'submitted')}
+					<h2>There are no submitted giveaways.</h2>
+				</div>;
+			}
+			return <div class="pad">
+				{makePageHeader(this.user, args[0] === 'add' ? 'submitted-add' : 'submitted')}
+				{(() => {
+					if (!adding) {
+						const buf = [];
+						for (let giveaway of giveaways) {
+							const claimCmd = giveaway.claimed === user.id ?
+								`/giveaway unclaim ${giveaway.targetUserID}` :
+								`/giveaway claim ${giveaway.targetUserID}`;
+							const claimedTitle = giveaway.claimed === user.id ?
+								"Unclaim" : giveaway.claimed ?
+									`Claimed by ${giveaway.claimed}` : `Claim`;
+							const disabled = giveaway.claimed && giveaway.claimed !== user.id ? " disabled" : "";
+							buf.push(<div class="infobox">
+								{(() => {
+									if (wifiData.submittedGiveaways.lottery.includes(giveaway as LotteryGiveawayData)) {
+										giveaway = giveaway as LotteryGiveawayData;
+										return <>
+											<h3 style={{textAlign: 'center'}}>Lottery</h3>
+											<hr />
+											<strong>Game:</strong> {gameName[giveaway.game]}, <strong>Giver:</strong> {giveaway.targetUserID},
+											&#32;<strong>OT:</strong> {giveaway.ot}, <strong>TID:</strong> {giveaway.tid},
+											&#32;<strong># of winners:</strong> {giveaway.winners}
+											{!!giveaway.claimed && <><br /><strong>Claimed:</strong> {giveaway.claimed}</>}<br />
+											<strong>Pok&eacute; Ball:</strong> <psicon item={giveaway.ball} />
+											<details>
+												<summary><psicon pokemon={giveaway.prize.species} /> Prize</summary>
+												{Giveaway.convertIVs(giveaway.prize, giveaway.ivs).replace(/\n/g, '<br />')}
```

Comment:
This is not going to do what you think it does.

Reply:
The replacer replaces linebreaks with the escaped text `<br />`, not an actual `br` element.

Reply:
this was a remnant of old code that I forgot to fix, i didn't intentionally do this